clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/ChangelogClientConfig.java

```diff
@@ -142,6 +144,15 @@ public ChangelogClientConfig setVersionSwapDetectionIntervalTimeInMs(long interv
     return this;
   }
 
+  public long getDatabaseSyncBytesInterval() {
+    return databaseSyncBytesInterval;
+  }
+
+  public ChangelogClientConfig setDatabaseSyncBytesInterval(long databaseSyncBytesInterval) {
```

Comment:
let's add a bit of javadoc here to specify what this is for.

Reply:
Sure, added.